src/cljs/main/broadfcui/page/workspace/data/copy_data_workspaces.cljs

```diff
@@ -30,30 +30,30 @@
                   {:header "Created By" :starting-width 200}
                   (table/date-column {})
                   {:header "Access Level" :starting-width 106}
-                  {:header "Protected" :starting-width 86
-                   :content-renderer #(if % "Yes" "No")}]
+                  {:header "Authorization Domain" :starting-width 150
+                   :content-renderer #(if % % "None")}]
         :toolbar (default-toolbar-layout
                   (let [num (:num-filtered props)]
                     (when (pos? num)
                       (str (:num-filtered props)
-                           " workspace(s) unavailable because they contain protected data."))))
+                           " workspace(s) unavailable because they contain data from other authorization domains."))))
         :data (:workspaces props)
         :->row (fn [ws]
                  [(get-in ws ["workspace" "namespace"])
                   ws
                   (get-in ws ["workspace" "createdBy"])
                   (get-in ws ["workspace" "createdDate"])
                   (ws "accessLevel")
-                  (get-in ws ["workspace" "isProtected"])])}]])})
+                  (get-in ws ["workspace" "authorizationDomain" "usersGroupName"])])}]])})
 
 (defn- remove-self [workspace-id workspace-list]
   (filter #(not= workspace-id {:namespace (get-in % ["workspace" "namespace"])
                                :name (get-in % ["workspace" "name"])}) workspace-list))
 
-(defn- filter-workspaces [this-realm workspace-list]
-  (filter #(let [src-realm (get-in % ["workspace" "realm" "groupName"])]
+(defn- filter-workspaces [this-authdomain workspace-list]
+  (filter #(let [src-authdomain (get-in % ["workspace" "authorizationDomain" "usersGroupName"])]
```

Comment:
**Before:** Workspace list in data import page shows only workspaces that are not protected by the dbGapAuthorizedUsers realm and, if the destination workspace was protected, workspaces that were protected by the realm.
**After:** Workspace list in data import shows all non protected workspaces and workspaces that have the same authorization domain as the destination workspace